src/pages/ProjectPage.tsx

```diff
@@ -112,7 +112,7 @@ const ProjectPage = () => {
     setFilteredTestRuns(
       testRuns.filter(
         (t) =>
-          t.name.includes(query) && // by query
+          t.name.toLowerCase().includes(query.toLowerCase()) && // by query
```

Comment:
could you make separate PR for this?
I'll need to create separate ticket to have it in release log

Reply:
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/173